packages/components/psammead-sitewide-links/src/Link/index.jsx

```diff
@@ -19,19 +19,21 @@ const StyleLinkText = styled.span`
   }
 `;
 
-const Link = ({ text, href, inline }) => (
-  <StyledLink inline={inline} href={href}>
+const Link = ({ text, href, inline, lang }) => (
```

Comment:
We should probably add a snapshot test for this component that verifies the `lang` attribute is being added.